blackbox/docs/sql/fulltext.txt

```diff
@@ -5,10 +5,10 @@
 Fulltext Search
 ===============
 
-In order to use fulltext searches on columns, one must create a
-:ref:`fulltext index with an analyzer <sql_ddl_index_fulltext>` on the related column
-upfront on table creation, see :ref:`fulltext-indices` for details.
-
+In order to use fulltext search on one or more columns, a
+:ref:`fulltext index with an analyzer <sql_ddl_index_fulltext>` has to be
+defined while creating the table. For more information
```

Comment:
I would rephrase to has to be defined on the column(s) when adding them to the table during its creation or altering (or alternation?)